transforms/create-element-to-jsx.js

```diff
@@ -0,0 +1,69 @@
+module.exports = function(file, api) {
+  var j = api.jscodeshift;
+  var root = j(file.source);
+
+  root.find(j.CallExpression, {callee: {object: {name: 'React'}, property: {name: 'createElement'}}})
```

Comment:
Ah, it does enforce trailing commas, but only if things are on multiple lines (which they will be)